pkg/features/kube_features.go

```diff
@@ -704,7 +705,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	CSIStorageCapacity:             {Default: false, PreRelease: featuregate.Alpha},
 	GenericEphemeralVolume:         {Default: false, PreRelease: featuregate.Alpha},
 	CSIVolumeFSGroupPolicy:         {Default: false, PreRelease: featuregate.Alpha},
-	RuntimeClass:                   {Default: true, PreRelease: featuregate.Beta},
+	RuntimeClass:                   {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.21
```

Comment:
I'm not sure what's the deprecation policy and if it's ok to remove it in 1.21. Should it be some later version?

Reply:
Is there a reason to keep it around? I mostly follow the pattern for other feature flags

Reply:
actually, don't we keep these around for 6 months, @liggitt  ?

Reply:
I followed an examples from this release:

https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L687-L688

https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L736

Should I fix them too?

Reply:
If they were GA'd in 20, they have to stay until 22.

Reply:
Fixed for this flag. I'll update other flags in separate PR